modules/manage/pages/iceberg/use-iceberg-catalogs.adoc

```diff
@@ -0,0 +1,113 @@
+
+To connect to a REST catalog, set the following cluster configuration properties:
+
+* config_ref:iceberg_catalog_type,true,properties/cluster-properties[`iceberg_catalog_type`]: `rest`
+* config_ref:iceberg_rest_catalog_endpoint,true,properties/cluster-properties[`iceberg_rest_catalog_endpoint`]: The endpoint URL for your Iceberg catalog, which you either manage directly, or is managed by an external catalog service.
+* config_ref:iceberg_rest_catalog_client_id,true,properties/cluster-properties[`iceberg_rest_catalog_client_id`]: The ID to connect to the REST catalog.
+* config_ref:iceberg_rest_catalog_client_secret,true,properties/cluster-properties[`iceberg_rest_catalog_client_secret`]: The secret data to connect to the REST catalog.
+
+For REST catalogs that use self-signed certificates, also configure these properties:
+
+* config_ref:iceberg_rest_catalog_trust_file,true,properties/cluster-properties[`iceberg_rest_catalog_trust_file`]: The path to a file containing a certificate chain to trust for the REST catalog.
+* config_ref:iceberg_rest_catalog_crl_file,true,properties/cluster-properties[`iceberg_rest_catalog_crl_file`]: The path to the certificate revocation list for the specified trust file.
+
+See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
+
+=== Example REST catalog configuration
+
+For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog:
+
+[,yaml]
+----
+iceberg_catalog_type: rest 
+iceberg_rest_catalog_endpoint: http://catalog-service:8181 
+iceberg_rest_catalog_client_id: <rest-connection-user>
+iceberg_rest_catalog_client_secret: <rest-connection-password>
+----
+
+And you use Apache Spark as a processing engine, configured to use a catalog named `streaming`:
+
+[,spark]
+----
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+# You may need to configure additional properties based on your object storage provider.
+# See https://iceberg.apache.org/docs/latest/spark-configuration/#catalog-configuration and https://spark.apache.org/docs/latest/configuration.html
+# For example, for AWS S3:
+# spark.sql.catalog.streaming.io-impl = org.apache.iceberg.aws.s3.S3FileIO
+# spark.sql.catalog.streaming.warehouse = s3://<bucket-name>/
+# spark.sql.catalog.streaming.s3.endpoint = http://<s3-uri>
```

Comment:
Need confirmation on whether these configs, definitions/placeholders, and additional docs are correct